### PR TITLE
Feature/cb2 11019

### PR DESCRIPTION
**Pull request by DanielFry-bjss**
## Description

Update VOTT pack tests to use V3 tech record service end points

Related issue: CB2-11019

## Before submitting (or marking as "ready for review")

- [x] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
- [x] Have you performed a self-review of the code
- [x] Have you have added tests that prove the fix or feature is effective and working
- [ ] Did you make sure to update any documentation relating to this change?

### CODE REVIEW

File: src/test/java/vott/updatestore/DynamoNOPDataPipelineTest.java [hunk 1]
```diff
@@ -115,83 +117,126 @@ public void setUp() throws Exception {
 
                 payloadPath = "src/main/resources/payloads/";
 
+                hgvTechRecordGen = new TechRecordHgvCompleteGenerator(new TechRecordHgvComplete());
+
                 psvTechRecordGen = new TechRecordPsvCompleteGenerator(new TechRecordPsvComplete());
 
+                lgvTechRecordGen = new TechRecordLgvCompleteGenerator(new TechRecordLgvComplete());
+
+                hgv_trg = new TechRecordHgvCompleteGenerator(new TechRecordHgvComplete());
```
Comment (FurkanErdogan-BJSS) [inline src/test/java/vott/updatestore/DynamoNOPDataPipelineTest.java:126]: I think I added this. But seems like we already have the same on line 120.

File: src/test/java/vott/updatestore/DynamoNOPDataPipelineTest.java [hunk 2]
```diff
@@ -206,46 +251,92 @@ public void twoTestType() {
         }
 
         @WithTag("Vott")
-        @Title("CB2-7553 - numberOfWheelsDriven accepted as string as well as string")
+        @Title("CB2-7553 - numberOfWheelsDriven accepted as string as well as number")
```
Comment (FurkanErdogan-BJSS) [inline src/test/java/vott/updatestore/DynamoNOPDataPipelineTest.java:254]: I should also update the test title.

**Review by FurkanErdogan-BJSS — CHANGES_REQUESTED** [review]

File: src/test/java/vott/updatestore/DynamoNOPDataPipelineTest.java (hunk 2 above)
Comment (DanielFry-bjss) [reply src/test/java/vott/updatestore/DynamoNOPDataPipelineTest.java]: title updated

File: src/test/java/vott/updatestore/DynamoNOPDataPipelineTest.java (hunk 1 above)
Comment (DanielFry-bjss) [reply src/test/java/vott/updatestore/DynamoNOPDataPipelineTest.java]: Removed hgv_trg and replaced usage with hgvTechRecordGen

**Review by FurkanErdogan-BJSS — APPROVED** [review]

**Pull request merged**